NAMESPACE

```diff
@@ -3,6 +3,8 @@
 export(AgeComp)
 export(BevertonHoltRecruitment)
 export(CreateTMBModel)
+export(SetFIMSFunctions)
```

Comment:
It looks like these were manually added to NAMESPACE. The workflow is to add the new exports to FIMS-package.R and run devtools::document(), which automatically generates the NAMESPACE. As is, the next time devtools::document() gets run, this file will be overwritten.

Reply:
Fixed with fc641ce8